experimental/standalone.go

```diff
@@ -52,6 +53,33 @@ func DoGRPC(id string, opts datasource.ServeOpts) error {
 	return datasource.Serve(opts)
 }
 
+// ManageGRPC is like DoGRPC but with automatic datasource instance management.
+func ManageGRPC(id string, instanceFactory datasource.InstanceFactoryFunc, opts datasource.ManageOpts) error {
```

Comment:
can we move this out of experimental?  so it lives next to Manage?

I *think* this should be our promoted way to develop plugins since it will have the best out-of-the-box dev experience

Reply:
Added separate pull request - https://github.com/grafana/grafana-plugin-sdk-go/pull/339 - if this one will be approved we can proceed with standalone separately.